frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -44,6 +44,7 @@ export default class extends React.Component<Props, State> {
   render() {
     const { account } = this.props;
     const { storagePaidAtBlockHash } = this.state;
+    const message = "from genesis" || "loading";
```

Comment:
This does not make any sense to me... Well, given ^ we should just remove this. I still wonder what this expression was meant to be.